homeassistant/components/calendar/__init__.py

```diff
@@ -230,6 +283,19 @@ async def async_get_events(
         """Return calendar events within a datetime range."""
         raise NotImplementedError()
 
+    async def async_create_event(self, **kwargs: Any) -> dict[str, Any]:
```

Comment:
If we only need the event uid from the return value, why should we return a dict?

Reply:
OK, reduced scope.

With regard to the websocket value, really, the uid is only used in tests at the moment and not in the frontend -- but tests could be made a little more complicated to query to get the uid.  I think it's probably Ok to leave, but just wanted to be fully transparent.

Reply:
I'd consider removing it, if we don't really need it.